packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -506,11 +506,11 @@ export default function (Commands, Cypress, cy, state, config) {
     }
 
     // if a screenshot has not been taken (by cy.screenshot()) in the test
-    // that failed and the command log is not hidden, we can bypass
+    // that failed and the runner is not hidden, we can bypass
     // UI-changing and pixel-checking (simple: true)
     // otherwise, we need to do all the standard checks
     // to make sure the UI is in the right place (simple: false)
-    const simple = !state('screenshotTaken') && !config('hideCommandLog')
+    const simple = !state('screenshotTaken') && !config('hideRunnerUi')
```

Comment:
`hideCommandLog` can still be set via `NO_COMMAND_LOG` right? Should that not still be taken into account here?

Reply:
So that was a change I made on the last PR, since we have the new `hideRunnerUi`, I decided to only have the new option control this behavior. So, `NO_COMMAND_LOG` will behave as it did before.